calc.go

```diff
@@ -458,6 +458,12 @@ func isOperatorPrefixToken(token efp.Token) bool {
 func (f *File) parseToken(sheet string, token efp.Token, opdStack, optStack *Stack) error {
 	// parse reference: must reference at here
 	if token.TSubType == efp.TokenSubTypeRange {
+		for _, definedName := range f.GetDefinedName() {
+			if definedName.Name == token.TValue {
+				token.TValue = definedName.RefersTo
```

Comment:
The scope of defined name should be checked here.

Reply:
It's no entirely clear to me which checks you would like to have?

The only thing I can think of is this:
I made a small xlsx file with 2 DefinedNames with the same name, one with Sheet scope the other with workbook scope. In this case Excel seems to calculate the value using the sheet scope, so presumably this should be the rule.  I have pushed the pull request accordingly. (Note: there was an additional change in sheet.go line 1424 needed, otherwise the workbook scope was lost)

Or do you have other/additional checks in mind?